### PR TITLE
Capstone Project: Rock Music Festival Website

### PR DESCRIPTION
**Pull request by Stephen-Adom**
# Capstone Project: Rock Music Festival Website
In this pull request, the following tasks were implemented:
- Design the home page for desktop and mobile screen
- Design the about page for desktop and mobile screen
- Home page have a link in the menu to the about page
- The logo in the header links to the home page.
- When the user clicks (or taps) the hamburger button on the header, the mobile menu appears over the page
- The mobile menu have a close (X) button that closes the menu
- The section "Featured speakers" created dynamically in JavaScript
- A video of project documentation showing the features of the website. Here is a video of the [ project overview](https://drive.google.com/file/d/1KpCxOiIFp4VCd45ukioY_-E8pc2Sdy_M/view?usp=sharing)

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -12,10 +17,161 @@
...
+              rock<span class="logo-name-orange">fest</span>
+            </h5>
+          </div>
+        </a>
+        <div class="collapse navbar-collapse" id="navbarSupportedContent">
+          <ul class="navbar-nav mb-2 mb-lg-0"></ul>
+          <a href="#" title="Buy ticket" class="navbar-btn">Buy Ticket</a>
+        </div>
+      </div>
+    </nav>
```
Comment (Qoosim) [inline index.html]: - [ ] Wonderful job so far. However, it seems you forgot to add the black top navbar to your project. The navbar consists of two parts. Black-background part and the white-background part. Kindly include the missing part to your project.

### Your Design
your-navbar

### Guiding Template 
temp-navbar

File: index.html [hunk 2]
```diff
@@ -12,10 +17,161 @@
...
+          </h5>
+        </div>
+        <div
+          class="d-flex align-items-start align-items-sm-start align-items-md-center align-items-lg-center flex-column flex-sm-column flex-md-row flex-lg-row gap-lg-2"
+        >
+          <p>2023 Rockfest by Designesia.</p>
+          <p>Some Rights Reserved.</p>
+        </div>
+      </div>
+    </footer>
```
Comment (Qoosim) [inline index.html]: - [ ] For the about footer page, the background-color is supposed to be black, not white.
- [ ] Also the parner section is not supposed to be displayed on the about footer page. Kindly take a look at both designs and adjust your design according to the guiding template.

### Your About footer page
your-about-footer

### Guiding template about footer page
temp-about-footer

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @Stephen-Adom, 

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Top Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GitHub Flow. :heavy_check_mark: 



## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

**Review by roshan-bajgain — APPROVED** [review]
Hi @Stephen-Adom, 

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 


Good luck! 🎉 


Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**